ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.test.js

```diff
@@ -40,7 +48,10 @@ describe('Secure Your Wallet Onboarding View', () => {
     expect(queryAllByText('Skip account security?')).toHaveLength(1);
   });
 
-  it('should not be able to click "skip" until "Skip Account Security" terms are agreed to', () => {
+  it('should not be able to click "skip" until "Skip Account Security" terms are agreed to', async () => {
+    const mockDispatch = jest.fn();
+    store.dispatch = mockDispatch;
```

Comment:
can we not mock the specific action being called?

Reply:
pushed a fix [here](https://github.com/MetaMask/metamask-extension/pull/16875/commits/2770e4c61a4b110549db7d2077ba3cd6a7d95804)